src/AddTeamMemberForm.jsx

```diff
@@ -14,21 +14,30 @@ class AddTeamMemberForm extends React.Component {
   }
   onChange(key) {
     // Return a curried function that sets a value to the key in this.state
-    return function(value) {
+    return (value) => {
```

Comment:
Per [this](http://eslint.org/docs/rules/arrow-parens) eslint rule, a parenthesis is "required," although it is acceptable syntax to omit it in this case.